### PR TITLE
mgmt, upgrade network to 2021-05 #25116

### PR DESCRIPTION
**Pull request by XiaofeiCao**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

Updated network `api-version` to `2021-05-01`.
close https://github.com/Azure/azure-sdk-for-java/issues/25116

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-
net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md [hunk 1]
```diff
@@ -2,13 +2,9 @@
 
 ## 2.10.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+### Dependency Updates
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:5]:
```suggestion
### Other Changes

#### Dependency Updates
```

**Review by weidongxu-microsoft — COMMENTED** [review]
Any manual changed code?

File: sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md (hunk 1 above)
Comment (haolingdong-msft) [inline sdk/resourcemanager/azure-resourcemanager-network/CHANGELOG.md:5]: Please put  'Dependency Updates' to 'Other Changes' category.
Refer here: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/resourcemanager/azure-resourcemanager-compute/CHANGELOG.md

**Comment by XiaofeiCao** [comment]
> Any manual changed code?

No manual changed code involved.

**Review by weidongxu-microsoft — APPROVED** [review]

**Review by haolingdong-msft — APPROVED** [review]

**Pull request merged**